artifactory/commands/generic/upload.go

```diff
@@ -100,7 +132,9 @@ func (uc *UploadCommand) upload() error {
 	var uploadParamsArray []services.UploadParams
 	// Create UploadParams for all File-Spec groups.
 	for i := 0; i < len(uc.Spec().Files); i++ {
-		uploadParams, err := getUploadParams(uc.Spec().Get(i), uc.uploadConfiguration)
+		file := uc.Spec().Get(i)
+		file.Props += syncDeleteProp
```

Comment:
Question: If *syncDeleteProp* is not empty, it always starts with a semi-column. Is this okay? (it may be okay, just making sure).

Reply:
It's okay.